toolchain/check/context.cpp

```diff
@@ -326,16 +334,26 @@ auto Context::LookupUnqualifiedName(Parse::NodeId node_id,
       scope_stack().LookupInLexicalScopes(name_id);
 
   // Walk the non-lexical scopes and perform lookups into each of them.
+  // Collect scopes to poison this name when it's found.
+  std::vector<LookupScope> scopes_to_poison;
```

Comment:
We prefer LLVM types over std types in most cases.

```suggestion
  llvm::SmallVector<LookupScope> scopes_to_poison;
```

Reply:
Done.